tests/utils.py

```diff
@@ -98,3 +98,21 @@ def assert_function_in_collapsed(function_name: str, collapsed: Mapping[str, int
     assert any(
         (function_name in record) for record in collapsed.keys()
     ), f"function {function_name!r} missing in collapsed data!"
+
+
+def get_python_version(application_docker_container: Container):
```

Comment:
can be a fixture, if you like can change

Reply:
Good idea